profilepage/script/script.js

```diff
@@ -0,0 +1,158 @@
+								<br>
+								<button id="smallFollowButton">Follow</button>
+							</div>
+						</div>
+					</div>`;
+	}
+	leftmidmid.innerHTML = div;
+}
+
+var tweetContent = function() {
+	var data = this.data;
+	var div = '';
+	var rightbottom = document.querySelector(".rightbottom");
+	for(var i in data) {
+		var div = div + `<div class="indposts">
+						<div class="ppimage">
+							<img src="${data[i].user.profile_img}"">
+						</div>
+						<div class="postcontent">
+							<div class="namehandle">
+								<span>${data[i].user.full_name}</span>
+								<span>@${data[i].user.user_name} &#8901</span>
+								<span>5m</span>
+								<span></span>
+							</div>
+							<div class="tweetbody">
+								<span>
+								${data[i].text}
+								</span>
+							</div>
+							<div class="postInteractions">
+								<div class="comments">
+									<img src="img/replies.png">
+									<span>${data[i].stats.comments}</span>
+								</div>
+								<div class="likes">
+									<img src="img/likes.png">
+									<span>${data[i].stats.likes}</span>
+								</div>
+								<div class="retweets">
+									<img src="img/retweets.png">
+									<span>${data[i].stats.retweets}</span>
+								</div>
+							</div>
+						</div>
+					</div>`;
+	}
+	rightbottom.innerHTML = div;
+}
+
+var followingContent = function() {
+	var data = this.data;
+	var div = '';
+	var followingCards = document.querySelector(".followingCardsContainer");
+	for(var i in data) {
+		var div = div + `
+				<div class="followingCards">
+					<div>
+						<div class="followingCardsBanner"><img src="${data[i].cover_img}"></div>
+						<div class="profilepic">
+							<img src="${data[i].profile_img}">
+							<div class="cardMeta">
+								<div class="userMeta">
+									<span>${data[i].full_name}</span>
+									<span>@${data[i].user_name}</span>
+								</div>
+							</div>
+						</div>
+					</div>
+					<span class="userBio">${data[i].user_bio}</span>
+				</div>`;
+	}
+	followingCards.innerHTML = div;
+}
+
+function querySelectorInnerHTML(target, data) {
+	var container = document.querySelector(target);
+	container.innerHTML = data;
+}
+
+function querySelectorImg(target, data) {
+	var container = document.querySelector(target);
+	container.src = data;
+}
+
+function querySelectorInnerHTMLAll(target, data, index) {
+	var container = document.querySelectorAll(target)[index];
+	container.innerHTML = data;
+}
+
+function dataConv(UTCSeconds) {
+	var date = new Date(UTCSeconds);
+	var months = [ "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+	var month = months[date.getMonth() + 1];
+	var day = date.getDate();
+	var year = date.getFullYear();
+	return day + " " + month + " " + year;
+}
+
+var mediaSquare = function() {
+	var photosAndVideos = document.querySelector(".photosAndVideos");
+	div = '';
+	for(var i = 0; i < 9; i++) {
+		var div = div + `<div class="mediaSquare"></div>`;
```

Comment:
This is where template strings come in. In most cases when you are joining two strings as above, you can replace it with template strings.

div = `${div} <div class="mediaSquare"></div>`;

Reply:
Try identifying other places where you could use template strings.